main.cpp

```diff
@@ -251,14 +252,15 @@ void MobilePhone::charge()
 
 bool MobilePhone::updateOperatingSystem(float osVersion)
 {
-    std::cout << "Updated Operating System to Version " << osVersion << std::endl;
+    std::cout << "Updated Operating System to Version " << osVersion << " on this " << MobilePhone::manufacturer << "-phone from " << MobilePhone::yearOfManufacture << std::endl;
```

Comment:
Don't use the fully-qualified name when accessing member variables inside member functions. 
You should only use the fully-qualified name when using **static** class member variables.
- [ ] revise